models/repo/repo_list.go

```diff
@@ -47,6 +47,14 @@ func (repos RepositoryList) Swap(i, j int) {
 	repos[i], repos[j] = repos[j], repos[i]
 }
 
+func (repos RepositoryList) GetRepoIDs() []int64 {
+	ids := make([]int64, len(repos))
+	for i := range repos {
+		ids[i] = repos[i].ID
+	}
+	return ids
```

Comment:
Why would this be better? List in => list out? Is it often that it happens that there are duplicates in the list? Not from the places where I use the function, because it are query results.

Reply:
@yp05327 Can I resolve this? I don't think a set is better.